plugins/base/src/test/kotlin/linkableContent/LinkableContentTest.kt

```diff
@@ -312,4 +312,66 @@ class LinkableContentTest : AbstractCoreTest() {
         }
 
     }
+
+    @Test
+    fun `Include module with description parted in two files`() {
```

Comment:
This IMHO is the wrong kind of test. You should really write a unit test for everything that is unit testable. 
I am not proposing to remove this test, but adding a proper unit test as well!

Reply:
But with this engine we are doing mainly functional tests and this one does not look much different than the others

Reply:
Yes the project is almost entirely relying on functional tests and this is not ideal! This particular component was designed to support unit tests. 

Maybe @Kordyjan can give further directions on this issue :)